### PR TITLE
Update ptcheat.rst

### PR DESCRIPTION
**Pull request by edwardleardi**
scheduler.step() should not be called at the start of every epoch as of [PyTorch 1.1.0](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate). Instead it should be called after the optimizer has updated the weights (after `optimizer.step()` is called)

### CODE REVIEW

**Comment by holly1238** [comment]
Hi @edwardleardi! Can you sign the CLA agreement so that I can merge the pull request? Thanks!
